.circleci/config.yml

```diff
@@ -150,7 +192,8 @@ workflows:
   version: 2
   main:
     jobs:
-      - build_test_publish:
+      - lint
+      - build-test-publish:
```

Comment:
With this change, it's possible for the `build-test-publish` job to publish a docker image even if lints fail. Since The `lint` job contains the package security checks, I think we need to fix this.

We could either make `lint` a dependency of `build-test-publish`, or we could split out `publish` into a separate job and make that depend on both `lint` and `build-test`.